src/main/java/edu/cornell/kfs/concur/batch/service/ConcurBatchUtilityService.java

```diff
@@ -0,0 +1,35 @@
+package edu.cornell.kfs.concur.batch.service;
+
+import java.sql.Date;
+
+import org.kuali.kfs.sys.batch.BatchInputFileType;
+
+import edu.cornell.kfs.concur.batch.xmlObjects.PdpFeedFileBaseEntry;
+
+public interface ConcurBatchUtilityService {
+    
+    String buildFullyQualifiedPdpOutputFileName(String paymentImportDirecotry, String pdpInputfileName);
+    
+    void createDoneFileForPdpFile(String fullyQualifiedPdpFileName);
```

Comment:
Adjusting method and parameter names as routine is not specific to PDP processing and could also be used by the collector.